src/views/Users/Login/Login.vue

```diff
@@ -164,31 +159,30 @@
         },
         methods: {
             ...mapActions('users', ['login', 'logout']),
-            logUser: async function(){
+            logUser: async function () {
```

Comment:
there's another much more stylish way of writing in case you are interested. I'd be great if we could harmonize all the methods.
` async logUser() { 
// code goes here
} ;`
I'm not expecting a modification here.

Reply:
@terazus I ll do all the changes you need in separated tickets later